cypress/utils/commands.js

```diff
@@ -23,6 +23,41 @@ export const CURRENT_TENANT = {
   },
 };
 
+// Overwrite default backend endpoint to customized one, remember set to original value after tests complete.
+export const currentBackendEndpoint = (() => {
+  let currentEndpoint = BACKEND_BASE_PATH;
+  const DEFAULT_ENDPOINT = BACKEND_BASE_PATH;
+  const REMOTE_NO_AUTH_ENDPOINT = Cypress.env('remoteDataSourceNoAuthUrl');
+
+  return Object.freeze({
+    DEFAULT: DEFAULT_ENDPOINT,
+    REMOTE_NO_AUTH: REMOTE_NO_AUTH_ENDPOINT,
+    /**
+     * Change current backend endpoint
+     * @param {*} changedEndPoint
+     * @param {*} immediately set immediately false to change tenant after all pending promise be invoked,
+     * useful for reset backend endpoint after all tests run.
+     */
+    set(changedEndPoint, immediately = true) {
```

Comment:
For ease of troubleshooting, could we add a `cy.log` inside the `set` function about the information in your comment code https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1229/files#diff-aba0ffbc791fcd54335abb5eecbfa9dbb4581b4c29df4fd3f41a7d0eca4af7d8R26

Reply:
The `cy.log` has been added to the `set` function.